app/views/kaminari/_paginator.html.haml

```diff
@@ -0,0 +1,18 @@
+-#  The container tag
```

Comment:
Comment should have a space after the `#`<br>7 consecutive comments can be merged into one